runtime/oti/vm_api.h

```diff
@@ -356,12 +356,14 @@ internalCreateArrayClass(J9VMThread* vmThread, J9ROMArrayClass* romClass, J9Clas
  * @param className String object representing name of the class to load
  * @param classLoader J9ClassLoader to use
  * @param options load options such as J9_FINDCLASS_FLAG_EXISTING_ONLY
+ * @param allowedBitsForClassName the allowed bits for a valid class name,
+ *        including CLASSNAME_VALID_NON_ARRARY, CLASSNAME_VALID_ARRARY, or CLASSNAME_VALID.
```

Comment:
INVALID should be documented here, with its meaning.

Reply:
Yes and `CLASSNAME_INVALID` was added given CLASSNAME_INVALID can be passed to findClassString.